### PR TITLE
Made cosmetic edits to 5.1 release notes.

### PR DESCRIPTION
**Pull request by nessita**
# Branch description
Review and cleanup of 5.1 release notes in preparation for the feature freeze happening on May 22nd.

# Checklist
- [X] This PR targets the `main` branch. 
- [X] The commit message is written in past tense, mentions the ticket number, and ends with a period.

### CODE REVIEW

File: docs/releases/5.1.txt [hunk 1]
```diff
@@ -244,16 +244,16 @@ Logging
...
+  each operation to highlight :class:`operation categories
   <django.db.migrations.operations.base.OperationCategory>`.
 
 Migrations
 ~~~~~~~~~~
 
 * The new ``Operation.category`` attribute allows specifying an
   :class:`operation category
-  <django.db.migrations.operations.base.OperationCategory>` used by the
+  <django.db.migrations.operations.base.OperationCategory>`, which is used by
```
Comment (sarahboyce) [inline docs/releases/5.1.txt:256]: I think I prefer this one the way it was 🤔

File: docs/releases/5.1.txt [hunk 2]
```diff
@@ -46,7 +46,7 @@ Minor features
 * The default iteration count for the PBKDF2 password hasher is increased from
   720,000 to 870,000.
 
-* In order to follow OWASP recommendations, the default ``parallelism`` of the
+* To follow OWASP recommendations, the default ``parallelism`` of the
   ``ScryptPasswordHasher`` is increased from 1 to 5.
```
Comment (sarahboyce) [inline docs/releases/5.1.txt:50]:
```suggestion
* The default ``parallelism`` of the ``ScryptPasswordHasher`` is 
  increased from 1 to 5, to follow OWASP recommendations.
```
(optional) maybe we switch the order?

**Review by sarahboyce — APPROVED** [review]
Thank you! These look lovely

File: docs/releases/5.1.txt (hunk 2 above)
Comment (nessita) [reply docs/releases/5.1.txt:50]: Applied.

File: docs/releases/5.1.txt [hunk 3]
```diff
@@ -26,6 +26,65 @@ only officially support the latest release of each series.
...
+to generate links that maintain existing query parameters while adding or
+changing specific ones.
+
+For instance, navigating pagination and query strings in templates can be
+cumbersome. Consider this template fragment that dynamically generates a URL
+for navigating to the next page within a paginated view:
+
+.. code-block:: html+django
+
+    {# Linebreaks added for readability, otherwise this would be one, long line. #}
```
Comment (sarahboyce) [inline docs/releases/5.1.txt:43]:
```suggestion
    {# Linebreaks added for readability, this should be one, long line. #}
```

**Review by sarahboyce — APPROVED** [review]

**Pull request merged**